extension/src/popup/views/SignAuthEntry/index.tsx

```diff
@@ -67,6 +68,25 @@ export const SignAuthEntry = () => {
     accountToSign,
   );
 
+  if (isHardwareWallet) {
+    return (
+      <ModalWrapper>
+        <WarningMessage
+          variant={WarningMessageVariant.warning}
+          handleCloseClick={() => window.close()}
+          isActive
+          header={t("Unsupported signing method")}
+        >
+          <p>
+            {t(
+              "Hardware wallets are currently not supported for signing arbitrary blobs.",
```

Comment:
The phrasing/ordering of words here _really_ throws me off for some reason :joy: suggestion:

```suggestion
              "Signing arbitrary data with a hardware wallet is currently not supported.",
```

Reply:
works for me, done in d1f4093f9203f7bba5a6395d1a19540fd02ef47b